wsd/wopi/StorageConnectionManager.cpp

```diff
@@ -150,3 +149,89 @@ StorageConnectionManager::getHttpSession(const Poco::URI& uri, std::chrono::seco
 
     return httpSession;
 }
+
+void StorageConnectionManager::initialize()
+{
+#if ENABLE_SSL
+    // FIXME: should use our own SSL socket implementation here.
+    Poco::Crypto::initializeCrypto();
+    Poco::Net::initializeSSL();
+
+    // Init client
+    Poco::Net::Context::Params sslClientParams;
+
+    // false default for upgrade to preserve legacy configuration
+    // in-config-file defaults are true.
+    SSLAsScheme = COOLWSD::getConfigValue<bool>("storage.ssl.as_scheme", false);
+
+    // Fallback to ssl.enable if not set - for back compatibility & simplicity.
+    SSLEnabled = COOLWSD::getConfigValue<bool>("storage.ssl.enable",
+                                               COOLWSD::getConfigValue<bool>("ssl.enable", true));
```

Comment:
Here we still use Poco? I used Botan (C++) once or maybe OpenSSL?

Reply:
Our sockets use OpenSSL. Where we use Poco (not many cases left), Poco's sockets do their own SSL (also using OpenSSL). This is the config for whether HTTPS is available for the storage communications.